gapic/api/artman_clouddebugger.yaml

```diff
@@ -0,0 +1,30 @@
+common:
+  api_name: google-clouddebugger-v2
+  proto_gen_pkg_deps:
+    - google-common-protos
+  import_proto_path:
+    - ${REPOROOT}/googleapis
+  src_proto_path:
+    - ${REPOROOT}/googleapis/google/devtools/clouddebugger/v2
+  service_yaml:
+    - ${REPOROOT}/googleapis/google/devtools/clouddebugger/clouddebugger.yaml
+  gapic_api_yaml:
+    - ${REPOROOT}/googleapis/google/devtools/clouddebugger/v2/clouddebugger_gapic.yaml
+  output_dir: ${REPOROOT}/artman/output
+  enable_batch_generation: True
+java:
+  final_repo_dir: ${REPOROOT}/google-cloud-java/google-cloud-clouddebugger
+python:
+  final_repo_dir: ${REPOROOT}/artman/output/gcloud-python-clouddebugger
+go:
+  final_repo_dir: ${REPOROOT}/gapi-clouddebugger-go
+csharp:
+  final_repo_dir: ${REPOROOT}/artman/output/gcloud-csharp-clouddebugger
+php:
+  final_repo_dir: ${REPOROOT}/artman/output/gcloud-php-clouddebugger
+ruby:
+  final_repo_dir: ${REPOROOT}/google-cloud-ruby/google-cloud-clouddebugger
+  skip_packman: True
```

Comment:
Offline conversation for the record:

* If the flag isn't set, and later they add a hand-written library on top, they just need to modify auto-generated package metadata files in the repo.
* If the flag isn't set, and a hand-written library already exists, those auto-generated files would need to be reverted manually before merging the generated files in the library's repo.

In both cases, one can go and set the flag after the fact so future runs don't recreate those files, without needing to restart the whole process.